### ISSUE TITLE
PUT agent/{id}/import for update the whole agent

### ISSUE THREAD

**Issue by dcalvom** [samtecspg/articulate#230]
### Feature Request

@vargasbo is currently exploring articulate and he wants to be able to manage his agent versions. Given that we are not managing the versions of the agent on articulate he will use GIT to store the agents and whenever he needs to use a specific version he would like to be able to import that version easily into Articulate but without having the need of creating an agent with a different name.

Currently, we just have a POST agent/import for adding a full agent with domains, entities, intents, scenarios, webhooks, etc...

The idea of the PUT agent/{id}/import is for updating the agent with the new payload so whenever you need to go back to an older version, or, whenever you made changes to your agent outside articulate you can update your agent without losing its current IDs and update only with the data provided in the payload.

**Comment by malave** [comment]
@dcalvom @wrathagom: I don't think it is hard to implement, given that we are only keeping the agent id and not any id for the rest of the models. If this is the case I would do something like this:

1. Find the Agent by its id
2. Update all attributes using the import file
3. Delete every domain, saying, action, etc. related to this agent
4. Insert new domains, sayings, actions, etc. using the import file

Is this the behavior you are expecting?
